src/utils/project-utils.ts

```diff
@@ -25,6 +25,28 @@ function getCurrentBallerinaProject(): Promise<BallerinaProject> {
     });
 }
 
+function getCurrentBallerinaProjectFromActiveEditor(): Promise<BallerinaProject> {
```

Comment:
Shall we use the same getCurrentBallerinaProject() method instead of duplicating the fucntionality?